docs/docs/cmd/spo/site/site-archive.mdx

```diff
@@ -0,0 +1,55 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo site archive
+
+Archives a site collection
+
+## Usage
+
+```sh
+m365 spo site archive [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --url <url>`
+: URL of the site collection.
+
+`-f, --force`
+: Don't prompt for confirmation.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command you must be a Global or SharePoint administrator.
+
+:::
+
```

Comment:
Let's add a note:

`To archive a hub site, you first need to unregister it as a hub site.`